config/config-network.yaml

```diff
@@ -134,7 +134,5 @@ data:
     # This can be used for making Knative report all URLs as "HTTPS" for example, if you're
     # fronting Knative with an external loadbalancer that deals with TLS termination and
     # Knative doesn't know about that otherwise.
+    # AutoTLS scheme semantics are left untouched.
     overrideExternalScheme: ""
```

Comment:
Since this only really changes the default in the noautotls case now, does it make sense to have this default to "http" and thereby avoid the if OverrideExternalScheme == "" bits in the code?

Reply:
If we go with the `default` you mentioned above, then yes! Imma think about it on my walk to lunch.